web-app/src/pages/test/color-demo.tsx

```diff
@@ -2,10 +2,11 @@ import React from "react";
 import styled, { createGlobalStyle } from "styled-components";
 
 import NoGlobalStyle from "src/components/no-global-style";
```

Comment:
@joshuatshaffer I think we should remove this file if we are no longer using it

Reply:
Good idea. I'll do that.